app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -329,10 +330,12 @@ public function requestActivation(string $id, RequestActivationRequest $request)
         $organization = OrganizationMapper::mapActivationRequest($request);
         $this->organizationRepository->save($organization);
 
-        $organizers = UdbOrganizerMapper::mapActivationRequest($request, $id);
-        $this->organizerRepository->create(...$organizers);
-
-        $this->integrationRepository->requestActivation(Uuid::fromString($id), $organization->id, $request->input('coupon'));
+        $this->integrationRepository->requestActivation(
+            Uuid::fromString($id),
+            $organization->id,
+            $request->input('coupon'),
+            new UdbOrganizers(...UdbOrganizerMapper::mapActivationRequest($request, $id))
```

Comment:
Is it possible to make `mapActivationRequest` return `UdbOrganizers`? (I forgot to press "add comment" for this one)

Reply:
I think it returns a single organization, so there is no need for a collection here.